filters.js

```diff
@@ -106,7 +106,9 @@ var filters = {
 };
 
 const CF_DATE_FORMAT = "YYYY-MM-DD"
-const ISO_STRING_FORMAT = "YYYY-MM-DDTHH:mm:ss.SSSZ"
+const CF_OBJECT_NUMERIC_KEY_MAP = {
+  CURRENCY: "value"
```

Comment:
Are we storing the keys of object that will contain a numeric value? Might want to add a comment

Reply:
Yeah we are to return a default value. Otherwise we don't know which attribute to update, since it won't be a number.
I'll add a comment